input/docs/running-builds/runners/dotnet-core-tool.md

```diff
@@ -0,0 +1,53 @@
+Order: 10
+Title: .NET Core Tool
+---
+
+:::{.alert .alert-info}
+This is the recommended way to run Cake Scripts.
+:::
+
+# Requirements
+
+The [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) NuGet package, is a .NET Core global tool compiled for .NET Core 2.1 or newer.
+
+# Bootstrapping for .NET Core Tool
+
+:::{.alert .alert-info}
+The following instructions require .NET Core 3.0 or newer.
+See [How to manage .NET Core tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools) for details and other options.
+:::
+
+## Setup
+
+There's a one-time setup required for configuring a repository to use Cake .NET Core tool.
+
+Make sure to have a tool manifest available in your repository or create one using the following command:
+
+```shell
+dotnet new tool-manifest
```

Comment:
@pascalberger not for this PR, but should we be explicit here and mention that the generated manifest file should be checked into source control, so that it can be used in subsequent builds?

Reply:
👍 I would like to add more tutorials which can show different aspects in depth, instead of just a single getting started. This would allow to document these kind of stuff. Added this point to https://github.com/cake-build/website/issues/179

Reply:
Sounds like a plan.